template-parts/content/content.php

```diff
@@ -9,23 +9,21 @@
  */
 
 ?>
-<article <?php post_class( 'entry' ); ?>>
-	<header class="entry__header">
-		<h1 class="entry__title"><?php the_title(); ?></h1>
-	</header>
+
+<article <?php post_class( 'wp-site-blocks wp-block-post-content' ); ?>>
```

Comment:
I think I described this too simply when I sent the example from Thought Catalog. I was just trying to get across the idea about making the rendered content match the alignment in the editor.

The child themes will probably still need the "header, content div, footer" pattern inside the article. 

This is more accurate.

```php
<article <?php post_class( 'wp-site-blocks wp-block-post-content' ); ?>>
	<header class="has-global-padding is-layout-constrained">
		<!-- header stuff like h1 -->
	</header>
	<div class="has-global-padding is-layout-constrained">
		<?php the_content(); ?>
	</div>
	<footer class="has-global-padding is-layout-constrained">
		<!-- footer stuff like author cards -->
	</footer>
</article>
```